data/json/recipes/other/power_supplies.json

```diff
@@ -763,6 +770,7 @@
     "skill_used": "mechanics",
     "time": "2 h 40 m",
     "autolearn": true,
+    "proficiencies": [ { "proficiency": "prof_intro_chemistry" } ],
```

Comment:
I think this and the next three `methanol_tank` recipes just simulate tediously refilling a tank with a syringe, based on what I read in the original PR. Not sure if you'd be learning anything, but it's really your call.

Reply:
Well, it is true that there are no reactions involved here. But methanol is a highly toxic chemical, so I think this activity can at least train a character safety measures and handling of chemical equipment (a syringe in this case).
I've assigned "Principles of Chemistry" to lots of such activities, and have given "(In)organic Chemistry" only to recipes in which actual reactions are taking place.

At least, that was my reasoning at the end of the process. Of course, the work started when lack of proficiencies actually penalized a character, so having un-learned "Principles of Chemistry" in a recipe was in fact worse than having no proficiency at all. It all changed with recent patches, so I understand that now some may see the addition as an attempt at aiding player (and we all know, player must suffer more :D , not less).

Reply:
That all sounds good to me! That division between proficiencies makes sense. 👍